bin/collator/src/local/service.rs

```diff
@@ -202,7 +197,7 @@ pub fn new_partial(
 /// Builds a new service.
 pub fn start_node<N>(
     config: Configuration,
-    #[cfg(feature = "evm-tracing")] evm_tracing_config: crate::evm_tracing_types::EvmTracingConfig,
+    evm_tracing_config: EvmTracingConfig,
```

Comment:
Perhaps the tracing config can just be moved under the existing `Configuration` struct?

Reply:
Configuration is part of sc_service

Reply:
Right, I mistook it with something else, my bad.